app/src/main/java/uk/co/neuralcubes/neuralates/ControlFragment.java

```diff
@@ -191,4 +230,29 @@ public String apply(Robot bender) {
             }
         });
     }
+
+    private void enableSpheroActions(){
```

Comment:
It comes down to a styling guide, I guess. I'll follow that convention from now on